apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-card.js

```diff
@@ -0,0 +1,107 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * Internal dependencies
+ */
+import './style-tour.scss';
+import PaginationControl from './pagination';
+import minimize from './icons/minimize';
+
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody, CardFooter, CardMedia, Flex } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+
+// import { useEffect, useState } from '@wordpress/element';
+
+function WelcomeTourCard( {
+	cardContent,
+	cardIndex,
+	lastCardIndex,
+	onMinimize,
+	onDismiss,
+	setCurrentCard,
+} ) {
+	const { description, heading, imgSrc } = cardContent;
+
+	return (
+		<Card className="welcome-tour-card" isElevated>
+			<CardOverlayControls onDismiss={ onDismiss } onMinimize={ onMinimize } />
+			<CardMedia>
+				<img alt="Editor Welcome Tour" src={ imgSrc } />
+			</CardMedia>
+			<CardBody>
+				<h2 className="welcome-tour-card__heading">{ heading }</h2>
+				<p className="welcome-tour-card__description">
+					{ description }
+					{ cardIndex === lastCardIndex ? (
+						<Button
+							className="welcome-tour-card__description"
+							isTertiary
+							onClick={ () => setCurrentCard( 0 ) }
+						>
+							Restart tour
```

Comment:
Shall we add in the `__()` call to translate these strings of text now, or did you want to hold off until later, if the words are still being decided on?

Reply:
We'll do `__( )` calls in a subsequent PR.

Reply:
Sounds good!